csharp/src/Ice/Internal/Protocol.cs

```diff
@@ -87,38 +89,13 @@ public sealed class Protocol
 
     internal static void checkSupportedEncoding(Ice.EncodingVersion v)
     {
-        if (v.major != encodingMajor || v.minor > encodingMinor)
+        if (v.major != Util.Encoding_1_1.major || v.minor > Util.Encoding_1_1.minor)
```

Comment:
This is referencing the wrong `Util` class `Ice.Internal.Util` instead of `Ice.Util`